src/components/tests/Button.test.js

```diff
@@ -0,0 +1,25 @@
+// dependencies
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen, cleanup } from '@testing-library/react';
+// local files
+import Button from '../Button';
+
+afterEach(() => {
+  cleanup();
+});
+
+describe('Button Component renders correctly', () => {
+  const buttonText = 'hello world';
+  render(<Button buttonText={buttonText} />);
+  screen.debug();
```

Comment:
remove this line if it is only used for local debugging

Reply:
Done